packages/core/admin/admin/src/components/ConfigurationsProvider/reducer.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable consistent-return */
+/*
+ *
+ * ConfigurationsProvider reducer
+ *
+ */
+
+import produce from 'immer';
+
+const initialState = {
+  menuLogo: {
+    logo: undefined,
```

Comment:
Maybe we can use `url` insead of `logo` as we are already in a logo context and it is more representative of the value
```suggestion
    url: undefined,
```

Reply:
Also can we use `null` instead of `undefined`?